### PR TITLE
* change content of field property-address-select

### PR DESCRIPTION
**Pull request by sulthan-ahmed**
* add dependencies to package.json express, hof-controllers, hof-form-controllers, hof-model, hof-template-misins, hogas.js, lodash, so-acceptance
* npm-shrinkwrap updated
* change const to let as it wasn't a const in address-lookup

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -35,13 +35,21 @@
   },
   "dependencies": {
     "browserify": "^12.0.1",
+    "express": "https://registry.npmjs.org/express/-/express-4.14.0.tgz",
```
Comment (lennym) [inline package.json]: I don't think you meant this.

Comment (sulthan-ahmed) [inline package.json]: hmm that's very strange I just did the usual `npm i express -save`

I'll sort it out

**Comment by sulthan-ahmed** [comment]
ok amended @lennym

**Comment by JoeChapman** [comment]
@sulthan-ahmed did dependencies ended up being defined as tarballs because shrinkwrap?

**Comment by sulthan-ahmed** [comment]
I still don't know why it happens @JoeChapman anytime I `use npm -i --save` or `--saveDev`

File: package.json [hunk 2]
```diff
@@ -35,13 +35,21 @@
...
+    "hof-model": "^2.1.0",
+    "hof-template-mixins": "^2.0.0",
     "hof-transpiler": "0.1.0",
+    "hogan.js": "^3.0.2",
     "homeoffice-countries": "^0.1.0",
     "jquery": "^3.1.1",
+    "lodash": "^4.16.3",
     "node-fetch": "^1.6.3",
     "node-sass": "^3.7.0",
+    "so-acceptance": "^4.2.2",
```
Comment (sulthan-ahmed) [inline package.json]: Good spot, I've updated. 

So for my learning the dependencies section should be those that are needed for production and devdependencies are just for development purposes.  To be aware of this when installing stuff DevDependencies I guess are mainly dependencies that are used for testing, linting, grunting and all that jazz

**Comment by JoeChapman** [comment]
@sulthan-ahmed they likely installed as tarballs because of shrinkwrap. 
`rm -rf shrinkwrap.json`
`npm i`
`npm shrinkwrap`

**Comment by JoeChapman** [comment]
@lennym are you happy with the changes?

**Comment by JoeChapman** [comment]
@sulthan-ahmed can you fix the linting errors please

**Comment by sulthan-ahmed** [comment]
thanks @JoeChapman

**Pull request merged**